internal/pkg/api/api.go

```diff
@@ -128,6 +129,14 @@ func (a *apiServer) UploadChunk(w http.ResponseWriter, r *http.Request, id strin
 	}
 }
 
+func (a *apiServer) GetFile(w http.ResponseWriter, r *http.Request, id string) {
+	zlog := hlog.FromRequest(r).With().Logger()
+	if err := a.ft.handleSendFile(zlog, w, r, id); err != nil {
+		cntFileDeliv.IncError(err)
```

Comment:
```suggestion
		cntFileDeliv.IncError(err)
		w.Header().Set("Content-Type", "application/json")
```
We should ensure that the content type header of error responses is set.